contrib/natss/pkg/dispatcher/dispatcher.go

```diff
@@ -182,25 +184,26 @@ func (s *SubscriptionsSupervisor) Connect(stopCh <-chan struct{}) {
 	}
 }
 
-func (s *SubscriptionsSupervisor) UpdateSubscriptions(channel *eventingv1alpha1.Channel, isFinalizer bool) error {
+// UpdateSubscriptions creates/deletes the natss subscriptions based on channel.Spec.Subscribable.Subscribers
```

Comment:
it would be nice to describe what it returns as well, now that it returns a map of failed subscriptions?